README.md

```diff
@@ -1,8 +1,23 @@
 # Transmogrifiers
 This repo contains the filters, standard library, and schema files to properly process public dataset json files retrieved from open data sets provided by municipalities.
 
-## Functionality
-The filter folder contains javascript files used to process json files for dataset standardization; these files are retrieved from open datasets. The output json is defined in the `schemas` folder.
+# Purpose
+ODEN Transmogrifiers is a tool that automates the process of data extraction and transformation. The Transmogrifier supports extracting data from diverse sources.
```

Comment:
I think this should say that we the provide the following (filters, schema, library, and collector) for the transmogrifier to combine and use (could maybe link to the transmog repo). We can use the original line for the transmogrifier repo though, with added information about sinks.